src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -403,6 +403,16 @@ public JobStatus permanentlyDeleteTickets(long id, long... ids) {
                 .set("query", searchTerm).set("section", sectionId), handleList(Article.class, "results"));
     }
 
+    public Iterable<Article> getArticleFromSomeLabels(List<String> labels) {
```

Comment:
Naming seems a little off to me, especially with the fact singular is used for returning a collection. How about something like..

getArticlesWithAllLabels
searchArticlesWithLabels

Reply:
I don't understand what is meant by "get" vs "search" if anything. The part I was looking to make clear was that one method requires at least one label to be found, the other requires all labels to be found. IMO, "WithAll" is good, but the idea of "with at least one" is not captured in the name "searchArticlesWithLabels". Not sure if "search" implies such behavior. You objected to the use of singular for returning a collection, I agree with that, I followed the existing pattern established by the `getArticleFromSearch` method. I think `getArticlesFromSomeLabels` and `getArticlesFromAllLabels` best captures the concerns, though I don't like the apparent API inconsistency with the singular Article vs Articles.

Reply:
How about getArticlesFromAllLabels and getArticlesFromAnyLabels

Reply:
Works for me, as long as everyone doesn't mind the inconsistency of Article plurality with `getArticleFromSearch`. Should I make the change, or will someone else do it?

Reply:
Or introduce corrected method with deprecated annotation on the singular.

Reply:
Who is the "gatekeeper" of the public-facing API? I'm trying to add functionality but not dictate API design in general. I feel like someone more qualified should handle the naming issue. I'm happy with getArticlesFromAllLabels and getArticlesFromAnyLabels. I'm going to make that change and hope it goes through.